popUp.css

```diff
@@ -0,0 +1,205 @@
+#overlay {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  background: #ebf0ee;
+  width: 100vw;
+  height: 108vh;
+  opacity: 0.9;
+  z-index: 100;
+  display: none;
```

Comment:
- [ ] Please from the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md), you should stick to the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=YHGkZAlalTNDuhZx-0) as much as possible. :smiley: Why don't you re-style your models by giving them `margin-top` and `margin-bottom` also that they can match the Figma design?

You can see the differences below. :-1: 

##### YOUR DESIGN
![image](https://user-images.githubusercontent.com/92303891/204070564-bced14c1-2164-47a5-911e-d58e522a9458.png)

##### FIGMA DESIGN
![image](https://user-images.githubusercontent.com/92303891/204070585-1ab72995-54ce-490c-88ee-9be58c4efe8e.png)